atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownDeleteTest.java

```diff
@@ -36,13 +36,13 @@ void testSetup(CassandraKeyValueService kvs) {
 
     @Test
     public void deletingThrows() {
-        assertThrowsInsufficientConsistencyExceptionAndDoesNotChangeCassandraSchema(
+        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(
```

Comment:
This seems odd - why do we need to change this? We should still be throwing InsufficientConsistencyExceptions here.